app/src/picovr/cpp/DeviceDelegatePicoVR.cpp

```diff
@@ -103,14 +106,18 @@ struct DeviceDelegatePicoVR::State {
     UpdatePerspective();
     UpdateEyeTransform();
 
+    gazeIndex = VRBrowserPico::GetGazeIndex();
+
     for (int32_t index = 0; index < kMaxControllerCount; index++) {
       controllers[index].index = index;
       if (index == 0) {
         controllers[index].hand = ElbowModel::HandEnum::Right;
       } else {
         controllers[index].hand = ElbowModel::HandEnum::Left;
       }
-      controllers[index].is6DoF = true;
+      if (index != gazeIndex) {
+        controllers[index].is6DoF = true;
+      }
```

Comment:
Not sure if it matters but I guess technically, the Neo2 gaze controller is 6DoF.

Reply:
Updated